content/docs/reference/pkg/_index.md

```diff
@@ -155,40 +159,42 @@ Each cloud vendor has a dedicated package for deploying resources to it:
-* [**Fastly** (`fastly`)](https://pkg.go.dev/github.com/pulumi/pulumi-fastly/sdk/go/fastly)
-* [**F5 BigIP** (`f5bigip`)](https://pkg.go.dev/github.com/pulumi/pulumi-f5bigip/sdk/go/f5bigip)
-* [**GitLab** (`gitlab`)](https://pkg.go.dev/github.com/pulumi/pulumi-gitlab/sdk/go/gitlab)
-* [**Kafka** (`kafka`)](https://pkg.go.dev/github.com/pulumi/pulumi-kafka/sdk/go/kafka)
-* [**Keycloak** (`keycloak`)](https://pkg.go.dev/github.com/pulumi/pulumi-keycloak/sdk/go/keycloak)
-* [**Linode** (`linode`)](https://pkg.go.dev/github.com/pulumi/pulumi-linode/sdk/go/linode)
-* [**Mailgun** (`mailgun`)](https://pkg.go.dev/github.com/pulumi/pulumi-mailgun/sdk/go/mailgun)
-* [**MySQL** (`mysql`)](https://pkg.go.dev/github.com/pulumi/pulumi-mysql/sdk/go/mysql)
-* [**New Relic** (`newrelic`)](https://pkg.go.dev/github.com/pulumi/pulumi-newrelic/sdk/go/newrelic)
-* [**Okta** (`okta`)](https://pkg.go.dev/github.com/pulumi/pulumi-okta/sdk/go/okta)
-* [**OpenStack** (`openstack`)](https://pkg.go.dev/github.com/pulumi/pulumi-openstack/sdk/go/openstack)
-* [**Packet** (`packet`)](https://pkg.go.dev/github.com/pulumi/pulumi-packet/sdk/go/packet)
-* [**PostgreSQL** (`postgresql`)](https://pkg.go.dev/github.com/pulumi/pulumi-postgresql/sdk/go/postgresql)
-* **RabbitMQ** (`rabbitmq`): Coming soon!
-* [**Rancher2** (`rancher2`)](https://pkg.go.dev/github.com/pulumi/pulumi-rancher2/sdk/go/rancher2)
-* [**SignalFX** (`signalfx`)](https://pkg.go.dev/github.com/pulumi/pulumi-signalfx/sdk/go/signalfx)
-* [**SpotInst** (`spotinst`)](https://pkg.go.dev/github.com/pulumi/pulumi-spotinst/sdk/go/spotinst)
+* [**AWS** (`aws`)](https://pkg.go.dev/github.com/pulumi/pulumi-aws/sdk/v2/go/aws)
+* [**Azure** (`azure`)](https://pkg.go.dev/github.com/pulumi/pulumi-azure/sdk/v3/go/azure)
+    * [**Azure Active Directory** (`pulumi_azuread`)](https://pkg.go.dev/github.com/pulumi/pulumi-azuread/sdk/v2/go/azuread)
+* [**Google Cloud** (`gcp`)](https://pkg.go.dev/github.com/pulumi/pulumi-gcp/sdk/v3/go/gcp)
+* [**Kubernetes** (`kubernetes`)](https://pkg.go.dev/github.com/pulumi/pulumi-kubernetes/sdk/v2/go/kubernetes)
+* [**Aiven** (`aiven`)](https://pkg.go.dev/github.com/pulumi/pulumi-aiven/sdk/v2/go/aiven)
+* [**Alibaba Cloud** (`alicloud`)](https://pkg.go.dev/github.com/pulumi/pulumi-alicloud/sdk/v2/go/alicloud)
+* [**CloudAMQP** (`cloudamqp`)](https://pkg.go.dev/github.com/pulumi/pulumi-cloudamqp/sdk/v2/go/cloudamqp)
+* [**Cloudflare** (`cloudflare`)](https://pkg.go.dev/github.com/pulumi/pulumi-cloudflare/sdk/v2/go/cloudflare)
+* [**HashiCorp Consul** (`consul`)](https://pkg.go.dev/github.com/pulumi/pulumi-consul/sdk/v2/go/consul)
+* [**Datadog** (`datadog`)](https://pkg.go.dev/github.com/pulumi/pulumi-datadog/sdk/v2/go/datadog)
+* [**DigitalOcean** (`digitalocean`)](https://pkg.go.dev/github.com/pulumi/pulumi-digitalocean/sdk/v2/go/digitalocean)
+* [**DNSimple** (`dnsimple`)](https://pkg.go.dev/github.com/pulumi/pulumi-dnsimple/sdk/v2/go/dnsimple) !!!
```

Comment:
What's with the `!!!` at the end of this line and the next?

Reply:
it was a marker where I had gotten to in the file 😆 - removed now - great catch!!